src/main/java/com/thevoxelbox/voxelsniper/brush/EraserBrush.java

```diff
@@ -18,7 +18,7 @@
 public class EraserBrush extends Brush {
 
     private static final Set<Material> EXCLUSIVE_MATERIALS = EnumSet.of(
-            Material.AIR, Material.STONE, Material.GRASS, Material.DIRT, Material.SAND, Material.GRAVEL, Material.SANDSTONE);
+            Material.AIR, Material.STONE, Material.GRASS_BLOCK, Material.DIRT, Material.SAND, Material.GRAVEL, Material.SANDSTONE, Material.DEEPSLATE);
```

Comment:
Changing the material name like this could break compatibility with older versions of MC

Reply:
I see the point of compatibility, but this works fine for 1.17, and I do not think we will support versions older then 1.17, especially because there have been versions that worked well on 1.16.

I do however have another suggestion: Shouldn't this also include other blocks that would be part of natural generation, such as ores and bedrock, diorite, andesite, granite, calcite, tuff, etc?
Or maybe we should redesign this to only erase everything above a certain block?

Reply:
I will quickly make a compatibility fix, then I will merge it, thank you for your contributions.